README.md

```diff
@@ -1,50 +0,0 @@
-[![CircleCI](https://circleci.com/gh/facebookresearch/theseus/tree/main.svg?style=svg)](https://circleci.com/gh/facebookresearch/theseus/tree/main)
-[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/facebookresearch/habitat-sim/blob/main/LICENSE)
-[![python 3.7, 3.8](https://img.shields.io/badge/python-3.7%20%7C%203.8-blue.svg)](https://www.python.org/downloads/release/)
-[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-green?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
-
-
-# Theseus
-
-A library for differentiable nonlinear optimization built on PyTorch to support constructing various problems in robotics and vision as end-to-end differentiable architectures.
-
-The current focus is on nonlinear least squares with support for sparsity, batching, GPU, and backward modes for unrolling, truncated and implicit, and sampling based differentiation. This library is in beta with expected full release in mid 2022.
-
-
-## Getting Started
```

Comment:
We should still include a root readme for installation etc and a readme for examples and scripts.